python-package/brainflow/utils.py

```diff
@@ -5,8 +5,6 @@
 
 
 def check_memory_layout_row_major(data):
-    if not data:
```

Comment:
i notice you removed this to fix ci.  another approach might be something like `if data is None or len(data) == 0:`.  i'm not as familiar with python as c++ to know what is normal.